src/content/docs/style-guide/writing-docs/writer-workflow/tech-writer-workflow.mdx

```diff
@@ -157,4 +157,15 @@ To create a branch on the docs-website repo:
 3. Click on **New Branch**
 4. You will be prompted to name your new branch. Descriptive names are best. It's a great way to quickly clue people in to what your work is all about. For example, if you are working on What’s New pages, you might name the branch **Whats-new-updates**.
 
-When you create a new branch, don't forget to add the Jira ticket's key (DOC-1234) to the branch name and the PR title. 
\ No newline at end of file
+When you create a new branch, don't forget to add the Jira ticket's key (DOC-1234) to the branch name and the PR title.
```

Comment:
Earlier in this doc, this is referred to as "Jira issue ID" which seems more clear to me than "key." Whichever you decide to use, please be consistent :-)